 * Serving Flask app "predAPI" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: off
/home/ubuntu/anaconda3/lib/python3.6/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
Using TensorFlow backend.
 * Running on http://172.31.31.247:2222/ (Press CTRL+C to quit)
2018-11-23 05:55:41.104478: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
/home/ubuntu/anaconda3/lib/python3.6/site-packages/sklearn/utils/validation.py:475: DataConversionWarning: Data with input dtype object was converted to float64 by MinMaxScaler.
  warnings.warn(msg, DataConversionWarning)
65.38.73.195 - - [23/Nov/2018 05:55:42] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
65.38.73.195 - - [23/Nov/2018 05:55:46] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
65.38.73.195 - - [23/Nov/2018 05:55:53] "[37mGET /api/?days=3&language=PHP HTTP/1.1[0m" 200 -
65.38.73.195 - - [23/Nov/2018 05:56:08] "[37mGET /api/?days=3&language=PHP HTTP/1.1[0m" 200 -
65.38.73.195 - - [23/Nov/2018 05:56:14] "[37mGET /api/?days=3&language=ss HTTP/1.1[0m" 200 -
65.38.73.195 - - [23/Nov/2018 05:56:20] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
65.38.73.195 - - [23/Nov/2018 06:45:00] "[33mGET /?days=3&language=Python HTTP/1.1[0m" 404 -
65.38.73.195 - - [23/Nov/2018 06:45:00] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
65.38.73.195 - - [23/Nov/2018 06:58:23] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
135.0.239.119 - - [23/Nov/2018 11:23:59] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
135.0.239.119 - - [23/Nov/2018 11:24:00] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
135.0.239.119 - - [23/Nov/2018 11:24:10] "[37mGET /api/?days=365&language=Python HTTP/1.1[0m" 200 -
135.0.239.119 - - [23/Nov/2018 11:24:41] "[37mGET /api/?days=60&language=Python HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 14:55:43] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 14:55:54] "[37mGET /api/?days=365&language=Python HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 14:57:51] "[37mGET /api/?days=500&language=Python HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 14:59:44] "[37mGET /api/?days=500&language=PHP HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 14:59:46] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 15:01:25] "[37mGET /api/?days=500&language=Javascript HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 15:02:29] "[37mGET /api/?days=500&language=PHP HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 15:02:49] "[37mGET /api/?days=500&language=PHP HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 15:59:46] "[37mGET /api/?days=500&language=Javascript HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:00:15] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
[2018-11-23 16:15:04,918] ERROR in app: Exception on /api/ [GET]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1092, in _run
    subfeed, allow_tensor=True, allow_operation=False)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3490, in as_graph_element
    return self._as_graph_element_locked(obj, allow_tensor, allow_operation)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3569, in _as_graph_element_locked
    raise ValueError("Tensor %s is not an element of this graph." % obj)
ValueError: Tensor Tensor("Placeholder:0", shape=(7, 40), dtype=float32) is not an element of this graph.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "predAPI.py", line 36, in getPredTill
    pred = predDays(args_days, args_lang)
  File "/home/ubuntu/predictTrend/Statistic-Lang/trendModel/predService.py", line 131, in predDays
    model = load_model(path + '/models/%smodel.h5' % lang)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/engine/saving.py", line 419, in load_model
    model = _deserialize_model(f, custom_objects, compile)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/engine/saving.py", line 287, in _deserialize_model
    K.batch_set_value(weight_value_tuples)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py", line 2470, in batch_set_value
    get_session().run(assign_ops, feed_dict=feed_dict)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 929, in run
    run_metadata_ptr)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1095, in _run
    'Cannot interpret feed_dict key as Tensor: ' + e.args[0])
TypeError: Cannot interpret feed_dict key as Tensor: Tensor Tensor("Placeholder:0", shape=(7, 40), dtype=float32) is not an element of this graph.
132.205.229.228 - - [23/Nov/2018 16:15:04] "[1m[35mGET /api/?days=500&language=Java HTTP/1.1[0m" 500 -
132.205.229.228 - - [23/Nov/2018 16:15:06] "[37mGET /api/?days=500&language=Java&callback=jQuery110203015954536932952_1542989697506&_=1542989697507 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:17:34] "[37mGET /api/?days=500&language=PHP&callback=jQuery1102004330764304567758_1542989849122&_=1542989849123 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:18:41] "[37mGET /api/?days=500&language=Java&callback=jQuery110206840665271053843_1542989917328&_=1542989917329 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:21:34] "[37mGET /api/?days=500&language=Javascript&callback=jQuery1102021948464003219792_1542990091770&_=1542990091771 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:22:17] "[37mGET /api/?days=500&language=Java&callback=jQuery1102021948464003219792_1542990091770&_=1542990091772 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:24:04] "[37mGET /api/?days=500&language=Java&callback=jQuery110204922496204033773_1542990238080&_=1542990238081 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:26:54] "[37mGET /api/?days=500&language=Java&callback=jQuery1102010799176696770063_1542990408887&_=1542990408888 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:29:15] "[37mGET /api/?days=500&language=Java&callback=jQuery1102010799176696770063_1542990408887&_=1542990408889 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:31:24] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:34:57] "[37mGET /api/?days=500&language=Java&callback=jQuery1102010799176696770063_1542990408887&_=1542990408890 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:39:41] "[37mGET /api/?days=500&language=Java&callback=jQuery11020435135891425255_1542991169670&_=1542991169671 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:40:09] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:40:26] "[37mGET /api/?days=500&language=Java&callback=jQuery110208841021655614827_1542991221860&_=1542991221861 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:41:22] "[37mGET /api/?days=500&language=Java&callback=jQuery1102047192233931729466_1542991275642&_=1542991275643 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:44:01] "[37mGET /api/?days=500&language=Java&callback=jQuery1102000465132004546942_1542991435132&_=1542991435133 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:45:09] "[37mGET /api/?days=500&language=Java&callback=jQuery110203740452767142286_1542991498314&_=1542991498315 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:48:08] "[37mGET /api/?days=500&language=Java&jsoncallback=jQuery110209723733890820745_1542991683010&_=1542991683011 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:48:54] "[37mGET /api/?days=500&language=Java&jsoncallback=jQuery110204271093062713165_1542991728548&_=1542991728549 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:49:16] "[37mGET /api/?days=500&language=Java&jsoncallback=jQuery110207362689050102205_1542991751797&_=1542991751798 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:49:32] "[37mGET /api/?days=500&language=Java&jsoncallback=jQuery110207082242073586826_1542991765470&_=1542991765471 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:51:35] "[37mGET /api/?days=500&language=PHP&jsoncallback=jQuery110202825367129640084_1542991891389&_=1542991891390 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:56:21] "[37mGET /api/?days=500&language=Java&jsoncallback=success_jsonpCallback&_=1542992175516 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:56:59] "[37mGET /api/?days=500&language=Java&jsoncallback=success_jsonpCallback&_=1542992175517 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 16:58:42] "[37mGET /api/?days=500&language=Java&jsoncallback=success_jsonpCallback&_=1542992175518 HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:00:38] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:00:42] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:07:45] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
101.227.139.173 - - [23/Nov/2018 17:09:57] "[33mGET / HTTP/1.0[0m" 404 -
132.205.229.228 - - [23/Nov/2018 17:12:14] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:17:55] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:23:09] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:23:35] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:23:42] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:25:22] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:25:51] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:28:09] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:34:55] "[37mGET /api/?days=3&language=java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:35:06] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:35:16] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:41:44] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 17:47:46] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 18:56:05] "[37mGET /api/?days=3&language=PHP HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 18:56:11] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 18:56:11] "[37mGET /api/?days=3&language=Javascript HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 18:59:25] "[37mGET /api/?days=3&language=Perl HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 18:59:29] "[37mGET /api/?days=3&language=Ruby HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 18:59:32] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 18:59:35] "[37mGET /api/?days=3&language=TypeScript HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 18:59:35] "[37mGET /api/?days=3&language=C++ HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 18:59:42] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 18:59:45] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:00:11] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:00:13] "[37mGET /api/?days=3&language=C++ HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:00:19] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:02:30] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:02:36] "[37mGET /api/?days=3&language=C++ HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:04:12] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:04:52] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:06:54] "[37mGET /api/?days=3&language=java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:07:00] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:09:10] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:10:46] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
132.205.229.40 - - [23/Nov/2018 19:14:19] "[37mGET /api/?days=3&language=PHP HTTP/1.1[0m" 200 -
132.205.229.40 - - [23/Nov/2018 19:14:31] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
[2018-11-23 19:14:38,043] ERROR in app: Exception on /api/ [GET]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "predAPI.py", line 36, in getPredTill
    pred = predDays(args_days, args_lang)
  File "/home/ubuntu/predictTrend/Statistic-Lang/trendModel/predService.py", line 144, in predDays
    pre=model.predict(dataSequence)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/engine/training.py", line 1169, in predict
    steps=steps)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/engine/training_arrays.py", line 294, in predict_loop
    batch_outs = f(ins_batch)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py", line 2715, in __call__
    return self._call(inputs)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py", line 2675, in _call
    fetched = self._callable_fn(*array_vals)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1439, in __call__
    run_metadata_ptr)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/framework/errors_impl.py", line 528, in __exit__
    c_api.TF_GetCode(self.status.status))
tensorflow.python.framework.errors_impl.FailedPreconditionError: Attempting to use uninitialized value dense_1/bias
	 [[{{node dense_1/bias/read}} = Identity[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"](dense_1/bias)]]
132.205.229.228 - - [23/Nov/2018 19:14:38] "[1m[35mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 500 -
[2018-11-23 19:14:38,137] ERROR in app: Exception on /api/ [GET]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1092, in _run
    subfeed, allow_tensor=True, allow_operation=False)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3490, in as_graph_element
    return self._as_graph_element_locked(obj, allow_tensor, allow_operation)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3569, in _as_graph_element_locked
    raise ValueError("Tensor %s is not an element of this graph." % obj)
ValueError: Tensor Tensor("Placeholder:0", shape=(7, 40), dtype=float32) is not an element of this graph.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "predAPI.py", line 36, in getPredTill
    pred = predDays(args_days, args_lang)
  File "/home/ubuntu/predictTrend/Statistic-Lang/trendModel/predService.py", line 131, in predDays
    model = load_model(path + '/models/%smodel.h5' % lang)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/engine/saving.py", line 419, in load_model
    model = _deserialize_model(f, custom_objects, compile)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/engine/saving.py", line 287, in _deserialize_model
    K.batch_set_value(weight_value_tuples)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py", line 2470, in batch_set_value
    get_session().run(assign_ops, feed_dict=feed_dict)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 929, in run
    run_metadata_ptr)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1095, in _run
    'Cannot interpret feed_dict key as Tensor: ' + e.args[0])
TypeError: Cannot interpret feed_dict key as Tensor: Tensor Tensor("Placeholder:0", shape=(7, 40), dtype=float32) is not an element of this graph.
132.205.229.228 - - [23/Nov/2018 19:14:38] "[1m[35mGET /api/?days=3&language=PHP HTTP/1.1[0m" 500 -
132.205.229.228 - - [23/Nov/2018 19:14:40] "[37mGET /api/?days=3&language=C++ HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:14:45] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:14:46] "[37mGET /api/?days=3&language=C++ HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:15:33] "[37mGET /api/?days=3&language=C++ HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:15:38] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:20:20] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:20:23] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:20:54] "[37mGET /api/?days=500&language=Python HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:39:40] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:46:37] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:47:28] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:47:35] "[37mGET /api/?days=3&language=C++ HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:47:51] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:47:54] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:47:57] "[37mGET /api/?days=3&language=PHP HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:48:01] "[37mGET /api/?days=3&language=JupyterNotebook HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:48:07] "[37mGET /api/?days=3&language=JupyterNotebook HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:48:26] "[37mGET /api/?days=3&language=Perl HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:48:55] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:49:10] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:49:47] "[37mGET /api/?days=3&language=C++ HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:51:04] "[37mGET /api/?days=3&language=JupyterNotebook HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:51:09] "[37mGET /api/?days=3&language=JupyterNotebook HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:51:15] "[37mGET /api/?days=3&language=Ruby HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:51:20] "[37mGET /api/?days=3&language=Perl HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:52:00] "[37mGET /api/?days=3&language=perl HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:52:06] "[37mGET /api/?days=3&language=Perl HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:54:24] "[37mGET /api/?days=2&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:54:29] "[37mGET /api/?days=2&language=Java HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:54:35] "[37mGET /api/?days=2&language=PHP HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:54:41] "[37mGET /api/?days=2&language=Java HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:54:45] "[37mGET /api/?days=2&language=C++ HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:54:59] "[37mGET /api/?days=3&language=C++ HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:55:12] "[37mGET /api/?days=2&language=TypeScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:55:17] "[37mGET /api/?days=2&language=JupyterNotebook HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:55:23] "[37mGET /api/?days=2&language=Python HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:55:28] "[37mGET /api/?days=2&language=Ruby HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:55:36] "[37mGET /api/?days=2&language=Perl HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:55:45] "[37mGET /api/?days=2&language=TypeScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:56:40] "[37mGET /api/?days=2&language=TypeScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:56:46] "[37mGET /api/?days=2&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:56:52] "[37mGET /api/?days=2&language=Python HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:56:56] "[37mGET /api/?days=2&language=Perl HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:57:01] "[37mGET /api/?days=2&language=JupyterNotebook HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:57:28] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:57:35] "[37mGET /api/?days=3&language=Perl HTTP/1.1[0m" 200 -
132.205.229.228 - - [23/Nov/2018 19:57:43] "[37mGET /api/?days=100&language=Perl HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:59:21] "[37mGET /api/?days=2&language=TypeScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 19:59:43] "[37mGET /api/?days=2&language=C++ HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:00:09] "[37mGET /api/?days=2&language=C++ HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:00:28] "[37mGET /api/?days=2&language=C HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:00:47] "[37mGET /api/?days=2&language=Ruby HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:00:51] "[37mGET /api/?days=2&language=Perl HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:00:56] "[37mGET /api/?days=2&language=TypeScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:01:02] "[37mGET /api/?days=2&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:01:43] "[37mGET /api/?days=500&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:03:58] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:04:06] "[37mGET /api/?days=500&language=Ruby HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:04:25] "[37mGET /api/?days=500&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:04:52] "[37mGET /api/?days=500&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:10:58] "[37mGET /api/?days=500&language=Perl HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:11:20] "[37mGET /api/?days=500&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:11:43] "[37mGET /api/?days=500&language=Ruby HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:12:08] "[37mGET /api/?days=500&language=Perl HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:52:22] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:52:59] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:57:30] "[37mGET /api/?days=500&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:57:37] "[37mGET /api/?days=500&language=Ruby HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:57:41] "[37mGET /api/?days=500&language=TypeScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 20:59:55] "[37mGET /api/?days=500&language=Perl HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 21:00:00] "[37mGET /api/?days=500&language=Python HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 21:00:35] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 21:04:56] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 21:05:03] "[37mGET /api/?days=500&language=Python HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 21:05:09] "[37mGET /api/?days=500&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 21:44:30] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 21:45:07] "[37mGET /api/?days=500&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [23/Nov/2018 21:45:12] "[37mGET /api/?days=500&language=TypeScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 13:21:39] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 13:21:48] "[37mGET /api/?days=500&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:02:59] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:03:04] "[37mGET /api/?days=500&language=Java HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:03:07] "[37mGET /api/?days=500&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:04:08] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:04:37] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:04:39] "[37mGET /api/?days=3&language=html HTTP/1.1[0m" 200 -
132.205.229.228 - - [24/Nov/2018 14:05:13] "[37mGET /api/?days=3&language=html HTTP/1.1[0m" 200 -
132.205.229.228 - - [24/Nov/2018 14:05:13] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
132.205.229.228 - - [24/Nov/2018 14:05:23] "[37mGET /api/?days=3&language=HTML HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:05:56] "[37mGET /api/?days=3&language=HTML HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:06:02] "[37mGET /api/?days=3&language=TypeScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:06:06] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:11:06] "[37mGET /api/?days=3&language=TypeScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:11:41] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:15:43] "[37mGET /api/?days=3&language=HTML HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:15:46] "[37mGET /api/?days=3&language=TypeScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:15:52] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 14:31:05] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
[2018-11-24 15:07:44,772] ERROR in app: Exception on /api/ [GET]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "predAPI.py", line 36, in getPredTill
    pred = predDays(args_days, args_lang)
  File "/home/ubuntu/predictTrend/Statistic-Lang/trendModel/predService.py", line 144, in predDays
    pre=model.predict(dataSequence)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/engine/training.py", line 1169, in predict
    steps=steps)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/engine/training_arrays.py", line 294, in predict_loop
    batch_outs = f(ins_batch)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py", line 2715, in __call__
    return self._call(inputs)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py", line 2675, in _call
    fetched = self._callable_fn(*array_vals)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1439, in __call__
    run_metadata_ptr)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/framework/errors_impl.py", line 528, in __exit__
    c_api.TF_GetCode(self.status.status))
tensorflow.python.framework.errors_impl.FailedPreconditionError: Attempting to use uninitialized value dense_1/bias
	 [[{{node dense_1/bias/read}} = Identity[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"](dense_1/bias)]]
54.204.228.184 - - [24/Nov/2018 15:07:44] "[1m[35mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 500 -
[2018-11-24 15:07:44,847] ERROR in app: Exception on /api/ [GET]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1092, in _run
    subfeed, allow_tensor=True, allow_operation=False)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3490, in as_graph_element
    return self._as_graph_element_locked(obj, allow_tensor, allow_operation)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3569, in _as_graph_element_locked
    raise ValueError("Tensor %s is not an element of this graph." % obj)
ValueError: Tensor Tensor("Placeholder:0", shape=(7, 40), dtype=float32) is not an element of this graph.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "predAPI.py", line 36, in getPredTill
    pred = predDays(args_days, args_lang)
  File "/home/ubuntu/predictTrend/Statistic-Lang/trendModel/predService.py", line 131, in predDays
    model = load_model(path + '/models/%smodel.h5' % lang)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/engine/saving.py", line 419, in load_model
    model = _deserialize_model(f, custom_objects, compile)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/engine/saving.py", line 287, in _deserialize_model
    K.batch_set_value(weight_value_tuples)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py", line 2470, in batch_set_value
    get_session().run(assign_ops, feed_dict=feed_dict)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 929, in run
    run_metadata_ptr)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1095, in _run
    'Cannot interpret feed_dict key as Tensor: ' + e.args[0])
TypeError: Cannot interpret feed_dict key as Tensor: Tensor Tensor("Placeholder:0", shape=(7, 40), dtype=float32) is not an element of this graph.
54.204.228.184 - - [24/Nov/2018 15:07:44] "[1m[35mGET /api/?days=3&language=HTML HTTP/1.1[0m" 500 -
54.204.228.184 - - [24/Nov/2018 15:08:16] "[37mGET /api/?days=3&language=HTML HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 15:08:18] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 15:08:32] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 15:10:52] "[37mGET /api/?days=3&language=TypeScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 15:10:58] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 15:13:14] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.204.228.184 - - [24/Nov/2018 15:13:25] "[37mGET /api/?days=3&language=TypeScript HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 15:20:13] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
[2018-11-24 15:23:58,634] ERROR in app: Exception on /api/ [GET]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1092, in _run
    subfeed, allow_tensor=True, allow_operation=False)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3490, in as_graph_element
    return self._as_graph_element_locked(obj, allow_tensor, allow_operation)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3569, in _as_graph_element_locked
    raise ValueError("Tensor %s is not an element of this graph." % obj)
ValueError: Tensor Tensor("Placeholder:0", shape=(7, 40), dtype=float32) is not an element of this graph.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "predAPI.py", line 36, in getPredTill
    pred = predDays(args_days, args_lang)
  File "/home/ubuntu/predictTrend/Statistic-Lang/trendModel/predService.py", line 131, in predDays
    model = load_model(path + '/models/%smodel.h5' % lang)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/engine/saving.py", line 419, in load_model
    model = _deserialize_model(f, custom_objects, compile)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/engine/saving.py", line 287, in _deserialize_model
    K.batch_set_value(weight_value_tuples)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py", line 2470, in batch_set_value
    get_session().run(assign_ops, feed_dict=feed_dict)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 929, in run
    run_metadata_ptr)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1095, in _run
    'Cannot interpret feed_dict key as Tensor: ' + e.args[0])
TypeError: Cannot interpret feed_dict key as Tensor: Tensor Tensor("Placeholder:0", shape=(7, 40), dtype=float32) is not an element of this graph.
54.167.0.38 - - [24/Nov/2018 15:23:58] "[1m[35mGET /api/?days=3&language=TypeScript HTTP/1.1[0m" 500 -
54.167.0.38 - - [24/Nov/2018 15:23:59] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 15:24:02] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 15:24:07] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 15:24:12] "[37mGET /api/?days=3&language=HTML HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 15:24:18] "[37mGET /api/?days=3&language=TypeScript HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 15:26:08] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 15:36:37] "[37mGET /api/?days=3&language=HTML HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 15:36:58] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 15:47:51] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 15:56:35] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 15:56:39] "[37mGET /api/?days=3&language=HTML HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 15:57:35] "[37mGET /api/?days=3&language=HTML HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 15:59:37] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 15:59:40] "[37mGET /api/?days=3&language=TypeScript HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 16:00:12] "[37mGET /api/?days=3&language=TypeScript HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 16:00:14] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 16:00:21] "[37mGET /api/?days=3&language=TypeScript HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 16:00:25] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 16:00:29] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 16:00:34] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 16:04:31] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 16:05:25] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 16:05:26] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 16:05:29] "[37mGET /api/?days=3&language=TypeScript HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 16:05:36] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 18:21:25] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 18:21:40] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 18:41:34] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 18:41:41] "[37mGET /api/?days=3&language=HTML HTTP/1.1[0m" 200 -
54.167.0.38 - - [24/Nov/2018 18:41:50] "[37mGET /api/?days=3&language=Python HTTP/1.1[0m" 200 -
198.108.67.48 - - [25/Nov/2018 18:25:59] "[33mGET / HTTP/1.1[0m" 404 -
66.240.192.138 - - [26/Nov/2018 04:44:26] code 400, message Bad request syntax ('SSH-2.0-paramiko_2.0.2')
66.240.192.138 - - [26/Nov/2018 04:44:26] "[1m[35mSSH-2.0-paramiko_2.0.2[0m" HTTPStatus.BAD_REQUEST -
71.6.158.166 - - [26/Nov/2018 09:17:15] code 400, message Bad request syntax ('SSH-2.0-paramiko_2.0.2')
71.6.158.166 - - [26/Nov/2018 09:17:15] "[1m[35mSSH-2.0-paramiko_2.0.2[0m" HTTPStatus.BAD_REQUEST -
5.101.66.61 - - [26/Nov/2018 09:33:16] code 400, message Bad request syntax ('SSH-2.0-libssh2_1.7.0')
5.101.66.61 - - [26/Nov/2018 09:33:16] "[1m[35mSSH-2.0-libssh2_1.7.0[0m" HTTPStatus.BAD_REQUEST -
135.0.239.119 - - [26/Nov/2018 12:34:45] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
135.0.239.119 - - [26/Nov/2018 12:34:45] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
135.0.239.119 - - [26/Nov/2018 12:35:02] "[37mGET /api/?days=3&language=Perl HTTP/1.1[0m" 200 -
54.159.160.145 - - [26/Nov/2018 12:37:00] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
135.0.239.119 - - [26/Nov/2018 12:37:56] "[33mGET /repository HTTP/1.1[0m" 404 -
54.159.160.145 - - [26/Nov/2018 12:44:17] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.159.160.145 - - [26/Nov/2018 15:17:25] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.240 - - [26/Nov/2018 15:17:42] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.240 - - [26/Nov/2018 15:59:34] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.159.160.145 - - [26/Nov/2018 16:07:24] "[37mGET /api/?days=3&language=HTML HTTP/1.1[0m" 200 -
54.163.94.123 - - [26/Nov/2018 16:33:02] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
54.163.94.123 - - [26/Nov/2018 16:42:23] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.163.94.123 - - [26/Nov/2018 16:57:13] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
5.101.66.61 - - [26/Nov/2018 20:55:09] code 400, message Bad request syntax ('SSH-2.0-libssh2_1.7.0')
5.101.66.61 - - [26/Nov/2018 20:55:09] "[1m[35mSSH-2.0-libssh2_1.7.0[0m" HTTPStatus.BAD_REQUEST -
132.205.229.240 - - [26/Nov/2018 21:05:00] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.240 - - [26/Nov/2018 21:05:51] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
132.205.229.240 - - [26/Nov/2018 21:06:53] "[37mGET /api/?days=3&language=Java HTTP/1.1[0m" 200 -
54.163.94.123 - - [26/Nov/2018 21:28:44] "[37mGET /api/?days=3&language=JavaScript HTTP/1.1[0m" 200 -
